.jenkins/pipelines/standalone/report-code-coverage.Jenkinsfile

```diff
@@ -0,0 +1,83 @@
+pipeline {
+    agent {
+        label 'Jenkins-Shared-DC2'
+    }
+    options {
+        timeout(time: 30, unit: 'MINUTES')
+    }
+    parameters {
+        string(name: "REPOSITORY", defaultValue: "deislabs")
+        string(name: "BRANCH", defaultValue: "main", description: "Branch to build")
+        string(name: "COMMIT_ID", description: "Short commit ID used to archive build resoures")
+        string(name: "BUILD_URL", description: "URL of the upstream build")
+    }
+    environment {
+        MYST_SCRIPTS =    "${WORKSPACE}/scripts"
+        JENKINS_SCRIPTS = "${WORKSPACE}/.jenkins/scripts"
+        LCOV_FILE =       "lcov-${COMMIT_ID}.info"
+        BUILD_INFO =      "${BUILD_URL == '' ? 'N/A' : BUILD_URL}"
+    }
+    stages {
+        stage('Init Config') {
+            steps {
+                sh """
+                   # Initialize dependencies repo
+                   ${JENKINS_SCRIPTS}/global/init-config.sh
+
+                   # Install global dependencies
+                   ${JENKINS_SCRIPTS}/global/wait-dpkg.sh
+                   ${JENKINS_SCRIPTS}/global/init-install.sh
+                   """
+            }
+        }
+        stage('Download code coverage report') {
+            steps {
+                azureDownload(
+                    downloadType: 'container',
+                    containerName: 'mystikos-code-coverage',
+                    includeFilesPattern: "${LCOV_FILE}",
+                    storageCredentialId: 'mystikosreleaseblobcontainer'
+                )
+            }
+        }
+        stage('Report code coverage') {
+            when {
+                expression { return fileExists(LCOV_FILE) }
```

Comment:
If the LCOV file didn't exist (i.e. azureDownload doesn't download any files and doesn't fail) then this stage will be skipped and the build would be considered successful (assuming all else is)

Reply:
True, I'll remove this `when` clause and let the job fail if the file doesn't exist